install.sh

```diff
@@ -544,6 +560,41 @@ if [[ ! -z "$BVTSDK_PASSWORD" ]]; then
 	fi
 fi
 
+# Decrypt and extract the Flycapture SDK for the Point Grey cameras
+if [[ "$INSTALL_FLYCAP" == "true" ]]; then
+	instlog "Decrypting and installing the Flycapture SDK"
+	curl -s https://raw.githubusercontent.com/uf-mil/installer/master/flycapture-2-2.11.3.121-amd64.tar.gz.enc | \
+	openssl enc -aes-256-cbc -d -pass file:<(echo -n $SDK_PASSWORD) | tar -xpzC /tmp
+
+	# If the SDK does not decrypt correctly due to an incorrect password, fail the installation
+	if [[ ! -d /tmp/flycapture-2-2.11.3.121-amd64 ]]; then
+		instwarn "Terminating installation due to incorrect password for the Flycapture SDK"
+		exit 1
+	fi
+fi
+
+# Install the (unlisted...) dependencies needed for the Flycapture SDK
+sudo apt-get install -qq libraw1394-11
+sudo apt-get install -qq libusb-1.0-0
+sudo apt-get install -qq libgtk2.0-0
+sudo apt-get install -qq libgtkmm-2.4-1v5
+sudo apt-get install -qq libglademm-2.4-1v5
+sudo apt-get install -qq libgtkmm-2.4-dev
+sudo apt-get install -qq libglademm-2.4-dev
+sudo apt-get install -qq libgtkglextmm-x11-1.2-dev
+
+# Install the Flycapture SDK based on the install script packaged with it
+cd /tmp/flycapture-2-2.11.3.121-amd64
```

Comment:
Ah, the effects of sleep deprivation. And almost, you missed the `; do` at the end of the line.